key,default_value
userAgent,"Boto3/1.12.15 Python/2.7.17 Linux/4.4.0-039049-Microsoft Botocore/1.15.15"
eventID,"355a8dfe-93c1-41b0-9f8f-d8bbd67a30cb"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: logs:DescribeQueries on resource: arn:aws:logs:us-east-1:811596193553:log-group:null:log-stream:"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"0.35.92.20"
eventName,"DescribeQueries"
eventSource,"logs.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"c90fd249-9aa2-453b-b66f-edd14ad91523"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2020-03-06T13:35:14Z"
